wagtail/wagtailforms/models.py

```diff
@@ -295,7 +295,15 @@ def process_form_submission(self, form):
 
     def send_mail(self, form):
         addresses = [x.strip() for x in self.to_address.split(',')]
-        content = '\n'.join([x[1].label + ': ' + text_type(form.data.get(x[0])) for x in form.fields.items()])
+        content = []
+        for field in form:
+            value = field.value()
+            if isinstance(value, list):
+                value = ', '.join(value)
+                if len(value) == 0:
```

Comment:
I'd say the original function was wrong then and the default of the get method should have been an empty string. If @gasman agrees you can probably just fix the test instead of treating empty lists as a special case.